yoti-sdk-api/src/main/java/com/yoti/api/client/docs/DocScanClientBuilder.java

```diff
@@ -0,0 +1,46 @@
+package com.yoti.api.client.docs;
+
+import java.util.ServiceLoader;
+
+import com.yoti.api.client.KeyPairSource;
+
+/**
+ * Builder used to create {@link DocScanClient}
+ */
+public abstract class DocScanClientBuilder {
+
+    public static final DocScanClientBuilder newInstance() {
+        ServiceLoader<DocScanClientBuilder> docScanClientBuilders = ServiceLoader.load(DocScanClientBuilder.class);
+        if (!docScanClientBuilders.iterator().hasNext()) {
+            throw new IllegalStateException("Cannot find any implementation of " + DocScanClientBuilder.class.getSimpleName());
+        }
+        return docScanClientBuilders.iterator().next();
+    }
+
+    /**
+     * Sets the SDK ID to be used in the {@code DocScanClient}
+     * on the builder
+     *
+     * @param sdkId the sdk id
+     * @return the builder
+     */
+    public abstract DocScanClientBuilder withSdkId(String sdkId);
```

Comment:
@jlmartyres do you think clients will understand this is "_Client_ SDK ID", or do you think we need to be explicit and call it `withClientSdkId`?

Reply:
I'd go with withClientSdkId, we refer to it similarly on Modal and it's how it's referenced in the hub too

Reply:
Fixed in 1203d1f